scripts/appbuilder

```diff
@@ -171,8 +171,8 @@ docker_network_remove()
                 for CONTAINER in $(docker network inspect --format '{{range $k, $v := .Containers}}{{print $k}}{{end}}' $NETWORK_NAME)
                 do
                     echo "Force removing $CONTAINER from $NETWORK_NAME"
-                    docker network disconnect -f $NETWORK_NAME $CONTAINER;
-                done;
+                    docker network disconnect -f $NETWORK_NAME $CONTAINER || true
```

Comment:
Is this a case of deleting it twice? Wondering if silently passing can cause leakage of this resource.

Reply:
Ah looks like you already answered my question in the teams chat